cmd/juju/status/caasformatter.go

```diff
@@ -16,6 +16,13 @@ type caasStatusFormatter struct {
 	isoTime        bool
 }
 
+type caasUnitFormatInfo struct {
+	unit            params.CAASUnitStatus
+	unitName        string
+	applicationName string
+	meterStatuses   map[string]params.MeterStatus
```

Comment:
it's fine that this is here but I doubt we'll implement metering in the prototype